pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "evergreen.py"
-version = "3.6.29"
+version = "3.6.30"
```

Comment:
Can we make this a minor version change according to [semver](https://semver.org/) (I know we've been misusing it)?